fe/fe-core/src/main/cup/sql_parser.cup

```diff
@@ -4578,7 +4578,8 @@ literal ::=
   | DECIMAL_LITERAL:l
   {: RESULT = new DecimalLiteral(l); :}
   | STRING_LITERAL:l
-  {: RESULT = new StringLiteral(l); :}
+  // for support mysql embedded quota e.g. SELECT "Pat O""Hanrahan & <Matthew Eldridge]""";
```

Comment:
`embedded quotes` ?

Reply:
Inside the quotation marks, 2 quotation marks indicate a quotation mark.

Reply:
Is the `embedded quota` a typo? Better to replace it with `embedded quotation` or `embedded quote`.